install-build-deps.sh

```diff
@@ -25,12 +25,5 @@ go install \
   k8s.io/code-generator/cmd/deepcopy-gen \
   k8s.io/code-generator/cmd/defaulter-gen \
   k8s.io/code-generator/cmd/informer-gen \
-  k8s.io/code-generator/cmd/lister-gen
-
-# TODO: metalinter is archived, we should switch to github.com/golangci/golangci-lint
-# Install metalinter
-# Managing all linters that gometalinter uses with dep is going to take
-# a lot of work, so we install all of those from the release tarball
-METALINTER_VERSION="3.0.0"
-curl --silent --location "https://github.com/alecthomas/gometalinter/releases/download/v${METALINTER_VERSION}/gometalinter-${METALINTER_VERSION}-${OSARCH}.tar.gz" \
-  | tar -x -z -C "${GOBIN}" --strip-components 1
+  k8s.io/code-generator/cmd/lister-gen \
+  github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
@martina-if this is related to the comment you have in `Makefile`, I don't have the permission to push newly created docker image.